src/Microsoft.ML.Data/EntryPoints/InputBase.cs

```diff
@@ -188,25 +187,26 @@ public static string FindColumn(IExceptionContext ectx, DataViewSchema schema, O
                 var roleMappedData = new RoleMappedData(view, label, feature, group, weight, name, custom);
 
                 RoleMappedData cachedRoleMappedData = roleMappedData;
-                Cache.CachingType? cachingType = null;
+                IFileHandle fileHandle = null;
```

Comment:
`fileHandle` appears to always be set to `null`, and never assigned a value.

Reply:
Good catch. I was using it in the CachingOptions.Disk switch case where I then removed.

Reply:
Opened https://github.com/dotnet/machinelearning/pull/2759 to address this